packages/mobile/src/components/audio-breakdown-drawer/AudioBreakdownDrawer.tsx

```diff
@@ -100,105 +29,135 @@ const messages = {
 }
 
 export const AudioBreakdownDrawer = () => {
-  const styles = useStyles()
+  const { accountBalance, connectedWalletsBalance, totalBalance } =
+    useAudioBalance()
 
-  const accountBalance =
-    useSelector(getAccountBalance, (a, b) => Boolean(a && b && a === b)) ??
-    (BigInt(0) as AudioWei)
+  const { data: connectedWallets = [] } = useConnectedWallets()
+  const connectedWalletsBalances = useWalletAudioBalances({
```

Comment:
Isn't there a use balances hook where you can pass an include associated flag?

Reply:
yeah but it doesn't include the individual wallet balances which we also need for this view.
the flag only specifies whether to include the associated wallet balances in the total balance that's returned

Reply:
we could but didn't wanna do that now